### ISSUE TITLE
AssemblerError: Could not find the library: ../../src/utils.s from the relative path from the file. 

### ISSUE THREAD

**Issue by chakpongchung** [hm-riscv/vscode-riscv-venus#72]
I am trying to use this debugger for the following working code with `.import` using relative path:

https://github.com/chakpongchung/cs61c-1/blob/master/proj2/tests/assembly/test_argmax.s

but I see the following error:

```
AssemblerError: Could not find the library: ../../src/utils.s from the relative path from the file. Parent file not found! This library was imported by Import(path=/home/tom/Dropbox/leetcode/python/su20-proj2-starter/tests/assembly/test_argmax.s, relative=false).

```

**Comment by m1n1** [comment]
Hello @chakpongchung, thanks for the issue.

Our current Version published on the Marketplace doesn't support imports/Multifile yet. This is because, like also mentioned here #62 our imports are not that much tested. Because the extension is actively used at our Univesity during the semester/term, also for graded assignments, we decided not to update during the semester.

Unfortunately the vscode marketplace doesn't allow for beta releases :( If you need imports right now your options would be:

- Clone this repository and build the extension yourself.
- Wait for the next release. 

The current Semester is almost over so we might release the update soon. I will post here again when i know more.

**Comment by chakpongchung** [comment]
@m1n1 Thanks! looking forward to it!

**Comment by PointBreaker** [comment]
> Hello @chakpongchung, thanks for the issue.
> 
> Our current Version published on the Marketplace doesn't support imports/Multifile yet. This is because, like also mentioned here #62 our imports are not that much tested. Because the extension is actively used at our Univesity during the semester/term, also for graded assignments, we decided not to update during the semester.
> 
> Unfortunately the vscode marketplace doesn't allow for beta releases :( If you need imports right now your options would be:
> 
> * Clone this repository and build the extension yourself.
> * Wait for the next release.
> 
> The current Semester is almost over so we might release the update soon. I will post here again when i know more.

Hello, I'm also using this extension debuging my CS61C project, and multi file support is needed for the project, but sadly I don't have experience on developing vscode extenstions, is there a tutorial or guide I can refer to on building this extenstion locally?thanks very much.

**Comment by m1n1** [comment]
Hello @chakpongchung and @PointBreaker i just released v1.6.0, which includes multifile/import support. Relative imports should work for both Linux and Windows (and MacOS although untested) and there is also support for absolute imports on Linux.

Note that other file operations are NOT supported, like reading and writing to files from an assembly programm! So not all examples from [https://github.com/chakpongchung/cs61c-1](https://github.com/chakpongchung/cs61c-1) will work.

**Comment by PointBreaker** [comment]
> Hello @chakpongchung and @PointBreaker i just released v1.6.0, which includes multifile/import support. Relative imports should work for both Linux and Windows (and MacOS although untested) and there is also support for absolute imports on Linux.
> 
> Note that other file operations are NOT supported, like reading and writing to files from an assembly programm! So not all examples from https://github.com/chakpongchung/cs61c-1 will work.

that's amazing! thank you for your contribution!
